src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -783,14 +778,14 @@ private async Task<FlexibleServerModel.Server> GetExistingPostgresqlService(stri
                 cts.Token);
         }
 
-        private async Task<ManagedCluster> GetExistingAKSClusterAsync(string aksClusterName)
+        private async Task<ContainerServiceManagedClusterResource> GetExistingAKSClusterAsync(string aksClusterName)
         {
-            return await subscriptionIds.ToAsyncEnumerable().SelectAwait(async s =>
+            return await subscriptionIds.ToAsyncEnumerable().Select(s =>
             {
                 try
                 {
-                    var client = new ContainerServiceClient(tokenCredentials) { SubscriptionId = s };
-                    return (await client.ManagedClusters.ListAsync(cts.Token)).ToAsyncEnumerable(client.ManagedClusters.ListNextAsync);
+                    var client = armClient.GetSubscriptionResource(new(s));
+                    return client.GetContainerServiceManagedClustersAsync(cts.Token);
```

Comment:
Seems like this should be awaited, so that the try/catch catches the exception?

Reply:
I was minimizing the changes. That method is suffixed Async because it returns a type that derives from IAsyncEnumerable. There's nothing to await (and it would be a compiler error to try). What the catch will catch are first stage errors like "server unreachable" or "authentication", etc.)